### PR TITLE
Add postcss-loader when postcssLoaderOptions is present

### PR DESCRIPTION
**Pull request by tamagokun**
Specifying `postcssLoaderOptions` should trigger `postcss-loader` to be included in webpack.

Example use case: This would allow configs to directly include postcss plugins and reduce the amount of config files in a project.

### CODE REVIEW

**Comment by ruslansavenok** [comment]
This is a very useful feature as some postcss plugins can't be used without webpack context and `postcss.config.js` doesn't allow to access it.

For example, this one -> https://github.com/jantimon/iconfont-webpack-plugin

File: packages/next-css/css-loader-config.js [hunk 1]
```diff
@@ -10,12 +10,12 @@ module.exports = (
   })
   let postcssLoader
 
-  if (postcssConfig) {
+  if (postcssConfig || postcssLoaderOptions.config) {
```
Comment (ruslansavenok) [inline packages/next-css/css-loader-config.js]: Would suggest to use `postcssLoaderOptions` here, then in `next.config.js` you can do:

```
withSass({
  postCssLoaderOptions: {
    sourceMap: true,
    plugins: loder => ([....])
  },
  sassLoaderOptions: {
    sourceMap: true,
  },
})
```

Comment (tamagokun) [reply packages/next-css/css-loader-config.js]: I didn't even realize that `plugins` was outside of the config object. Yeah, definitely to change this up.

**Comment by tamagokun** [comment]
@ruslansavenok this should be good now.

**Comment by ruslansavenok** [comment]
@tamagokun seems to be. But I'm not affiliated with `zeit` and don't have permissions to merge this PR :)

**Comment by tamagokun** [comment]
darn 🤔

**Comment by tamagokun** [comment]
Fixed conflicts :+1:

**Comment by jescalan** [comment]
Hey just want to add my support here -- it's a very small and simple fix and would be very useful.

**Comment by timneutkens** [comment]
I'm going to close this in favor of https://github.com/zeit/next.js/issues/8626

Thanks for your contribution 🙏

**Pull request closed without merging**